lib/Valetudo.js

```diff
@@ -132,4 +132,24 @@ Valetudo.VACUUM_MODEL_PROVIDER = function() {
     }
 };
 
+Valetudo.prototype.shutdown = function() {
+    console.info("Valetudo shutdown in progress...")
+    
+    //shutdwon mqtt client
+    if(this.configuration.get("mqtt") && this.configuration.get("mqtt").enabled === true) {
```

Comment:
Instead of checking this, you could just call it every time and extend the MqttClient shutdown method to only act if there really is a connection

Reply:
A check is necessary because the mqttClient instance is only available if it is configured in the config. If I would simply call this.mqttClient every time I would get a NullPointer/NullReference (or whatever the javascript error is) every time, when no mqtt is used.
To make things easier it can also be checked by `if(this.mqqtClient)...` (changed this in the second commit).